src/api/storage-pools.tsx

```diff
@@ -88,10 +116,9 @@ const getMemberPool = (
   pool: Partial<LxdStoragePool>,
 ): Partial<LxdStoragePool> => {
   const poolForMember = { ...pool };
-  // config keys for ceph storage pool cannot be member specific
-  if (pool.driver === cephDriver) {
-    delete poolForMember.config;
-  }
```

Comment:
Ceph case is missing in the changed implementation

Reply:
this relates to the other comment for `getClusterPool`